src/wpt/harness.ts

```diff
@@ -71,9 +76,12 @@ declare global {
   var errors: Error[];
   // eslint-disable-next-line no-var -- https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#type-checking-for-globalthis
   var testOptions: TestRunnerOptions;
-
   // eslint-disable-next-line no-var -- https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#type-checking-for-globalthis
   var GLOBAL: { isWindow(): boolean };
+  // eslint-disable-next-line no-var -- https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#type-checking-for-globalthis
```

Comment:
Can you disable no-var in the beginning of this file rather than every single line?